config/locales/fra.yml

```
@@ -82,47 +88,60 @@ fra:
+        name: 'Fièvre'
+        name_gsm_7: 'Fievre'
+        notes: 'Sensation de fébrilité ou température mesurée supérieure ou égale à 38 °C (100,4 °F)'
       headache:
-        name: 'Maux de tete'
+        name: 'Maux de tête'
+        name_gsm_7: 'Maux de tete'
       muscle-pain:
         name: 'Douleurs musculaires'
       nausea-or-vomiting:
-        name: 'Nausees ou vomissements'
+        name: 'Nausées ou vomissements'
+        name_gsm_7: 'Nausees ou vomissements'
       new-loss-of-smell:
-        name: 'Nouvelle perte de l''odorat'
+        name: "Nouvelle perte de l'odorat"
       new-loss-of-taste:
-        name: 'Nouvelle perte de gout'
+        name: 'Nouvelle perte de goût'
+        name_gsm_7: 'Nouvelle perte de gout'
       pulse-ox:
-        name: 'd''oxymetrie de pouls'
-        notes: 'Saisissez votre mesure d''oxymetrie de pouls la plus faible au cours des dernieres 24 heures'
+        name: "d'oxymétrie de pouls"
+        name_gsm_7: "d'oxymetrie de pouls"
+        notes: "Saisissez votre mesure d'oxymétrie de pouls la plus faible au cours des dernières 24 heures"
       repeated-shaking-with-chills:
-        name: 'Tremblements repetes accompagnes de frissons'
+        name: 'Tremblements répétés accompagnés de frissons'
+        name_gsm_7: 'Tremblements repetes accompagnes de frissons'
       shortness-of-breath:
         name: 'Essoufflement'
       sore-throat:
         name: 'Maux de gorge'
       temperature:
-        name: 'Temperature'
-        notes: 'Veuillez entrer votre temperature actuelle en degres Fahrenheit'
+        name: 'Température'
+        name_gsm_7: 'Temperature'
+        notes: 'Veuillez entrer votre température actuelle en degrés Fahrenheit'
       used-a-fever-reducer:
-        name: 'Utilisation d''un agent pour diminuer la fievre'
-        notes: 'Avez-vous utilise un medicament pour diminuer la fievre au cours des dernieres 24 heures?'
+        name: "Utilisation d'un agent pour diminuer la fièvre"
+        name_gsm_7: "Utilisation d'un agent pour diminuer la fievre"
+        notes: 'Avez-vous utilisé un médicament pour diminuer la fièvre au cours des dernières 24 heures?'
       other:
         name: 'Other'
-        notes: 'J''aimerais que quelqu''un m''appelle pour faire le suivi d''autres symptomes ou d''autre chose'
-      no-symptoms: 'Aucun symptome'
+        notes: "J'aimerais que quelqu'un m'appelle pour faire le suivi d'autres symptômes ou d'autre chose"
+      no-symptoms: 'Aucun symptôme'
     threshold-op:
       less-than: 'moins que'
       less-than-or-equal: 'inferieur ou egal a'
       greater-than: 'plus grand que'
-      greater-than-or-equal: 'Plus grand ou egal a'
-      equal: 'egal a'
-      not-equal: 'pas egal a'
+      greater-than-or-equal: 'Plus grand ou égal à'
+      greater-than-or-equal_gsm_7: 'Plus grand ou egal a'
+      equal: 'égal à'
+      equal_gsm_7: 'egal a'
+      not-equal: 'pas égal à'
+      not-equal_gsm_7: 'pas egal a'
```

Now these are mixed case. Please change to all kebab case for all symptoms and threshold-ops. Leave the `twilio` section as snake case. And please make sure all code that indexes into these translations has the correct casing.
```suggestion
      not-equal-gsm-7: 'pas egal a'
```

---

good catch: https://github.com/SaraAlert/SaraAlert/pull/1171/commits/684028f8dd0425ffa369ee778281036cbe5b3f2a